hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/StringUtils.java

```diff
@@ -105,12 +105,12 @@ public static String toStartupShutdownString(String prefix, String... msg) {
   }
 
   public static void startupShutdownMessage(VersionInfo versionInfo,
-      Class<?> clazz, String[] args, Logger log) {
+      Class<?> clazz, String[] args, Logger log, String conf) {
```

Comment:
I think it would be better to pass a `Map` here instead of a fully formatted string.  This would allow filtering some sensitive properties if needed, as well as sorting the config properties alphabetically.  (Currently order is random, since `Properties` is just a `Hashtable`.)

Reply:
Passing the configuration object as it is and I added a `processForLogging` utility function to process the configuration object before logging it.